proposals/lip-0067.md

```diff
@@ -243,31 +238,31 @@ Key pair derived from the secret recovery phrase above, and the path `m/44'/134'
 ```json
 {
   "crypto": {
-    "version": "1",
-    "ciphertext": "086a59889e0e311422eeb15bb6c753aeead210c4494eb37cf7b8f01b0ed372d64e6a08cc77e0bc8170f79f199e2ce7b4c47fe5353e97e67d53c846c029c6cd08",
-    "mac": "9497dd4a84f05c941b22df1cce0cb7558fb3bdd66481c462d63e003dab837c7c",
-    "cipher": "aes-256-gcm",
+    "cipher": "aes-128-gcm",
     "cipherparams": {
-      "iv": "aa97507e9f8574b2e7c7ba8b",
-      "tag": "4b3362626c82b0ba1b2de62fb84a1e73"
+    "iv": "da7abd52538d936db5eb95d7b2b48cca",
+    "tag": "9131d1737229bba12edbc0de1fe2c1f1"
     },
+    "ciphertext": "55b03a67428dc9d43331221c55775ead6cf4a5575dfc37d6d72a59750d92d6cd803788ef905d48ce08789ec53198ef08c5b82af0dac100ca07f9ecbad361974552fd8c9aed8c8fb6f65ddafa6efba837",
     "kdf": "argon2id",
     "kdfparams": {
       "parallelism": 4,
       "iterations": 1,
-      "memory": 2024,
-      "salt": "12209d2c085ccbe40c09bb5a3ec7cefb"
-    }
+      "memory": 2048,
+      "salt": "9cf89f89fead59b4f03a1164b47bed2f"
+    },
+    "mac": "87d989d9681446268c70aa8d22474013220363bfabab6f4d06fcf31e6d9ebfef"
   },
+  "id": "ef52c117-d7cc-4246-bc9d-4dd506bef82f",
+  "version": "3",
   "metadata": {
     "name": "my lisk account",
     "description": "ed25519 key pair",
```

Comment:
Do you mean the examples are inconsistent in the two table?

> But the table above is using Ed25519 private key and here we write ed25519 key pair.

I am not sure I get it. We are deriving the key pair from Ed25519.

Reply:
I meant in the table in line 131 we specify the value "Ed25519 private key", but here we use "ed25519 key pair". I'm not sure if this property is only read by humans or also by machines. Just in case, I would use the specified value from the table.

Reply:
Okay. I have updated the example with "description: Ed25519 private key".